### PR TITLE
feat(deployed-addresses): add Mode Mainnet and Sepolia addresses

### PR DESCRIPTION
**Pull request by arwer13**
Based on https://research.lido.fi/t/wsteth-deployment-on-mode/7365

### CODE REVIEW

File: docs/deployed-contracts/index.md
Comment (TheDZhon) [inline docs/deployed-contracts/index.md:1]: I think that a new emergency msig is missing here :thinking:

File: docs/deployed-contracts/sepolia.md [hunk 1]
```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
   - L2LidoGateway
   - L2WstETHToken
 
+### Mode
+
+##### Ethereum part
+
+- L1ERC20TokenBridge: [`0xD847780c8808fc99727001C43E6C17A198b39729`](https://sepolia.etherscan.io/address/0xD847780c8808fc99727001C43E6C17A198b39729) (proxy)
```
Comment (TheDZhon) [inline docs/deployed-contracts/sepolia.md:107]: should be [0x16B929D35B200EA0ae0B93EABc3Bf9Ad611BF18F](https://sepolia.etherscan.io/address/0x16B929D35B200EA0ae0B93EABc3Bf9Ad611BF18F#readContract)

File: docs/deployed-contracts/sepolia.md [hunk 2]
```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
...
+- L1ERC20TokenBridge: [`0x514Aa1d41d840af7572509D75354A8612594eB30`](https://sepolia.etherscan.io/address/0x514Aa1d41d840af7572509D75354A8612594eB30) (impl)
```
Comment (TheDZhon) [inline docs/deployed-contracts/sepolia.md:108]: should be https://sepolia.etherscan.io/address/0x8c1E68A74E71594925c7D7a78Ef43657aec4d599

File: docs/deployed-contracts/sepolia.md [hunk 3]
```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
...
+
+##### Mode part
+
+- WstETH ERC20Bridged: [`0x953757Db814392642e614DaBba794Ef4381D8292`](https://sepolia.explorer.mode.network/address/0x953757Db814392642e614DaBba794Ef4381D8292) (proxy)
```
Comment (TheDZhon) [inline docs/deployed-contracts/sepolia.md:112]: should be [0x2C937931B5d544E3dff1d9D78c80d9772B55837A](https://sepolia.explorer.mode.network/address/0x2C937931B5d544E3dff1d9D78c80d9772B55837A)

File: docs/deployed-contracts/sepolia.md [hunk 4]
```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
...
+- WstETH ERC20Bridged: [`0x9C7CDB9312ddA72ACc24E4A3bE5aA6afeca2AAFE`](https://sepolia.explorer.mode.network/address/0x9C7CDB9312ddA72ACc24E4A3bE5aA6afeca2AAFE) (impl)
```
Comment (TheDZhon) [inline docs/deployed-contracts/sepolia.md:113]: should be https://sepolia.explorer.mode.network/address/0xC9C796E4CefdD38Eb462E84246bE41d500149162

File: docs/deployed-contracts/sepolia.md [hunk 5]
```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
...
+- L2ERC20TokenBridge: [`0x2F3a332eff5f45cEA0486464fa4080fcF52Ea876`](https://sepolia.explorer.mode.network/address/0x2F3a332eff5f45cEA0486464fa4080fcF52Ea876) (proxy)
```
Comment (TheDZhon) [inline docs/deployed-contracts/sepolia.md:114]: should be [0xd41a90e55bcfC1CbF96D78aE80BbCB56A6BA0008](https://sepolia.explorer.mode.network/address/0xd41a90e55bcfC1CbF96D78aE80BbCB56A6BA0008)

File: docs/deployed-contracts/sepolia.md [hunk 6]
```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
...
+- L2ERC20TokenBridge: [`0x2AD850d0c6b1846F9bBD8144362A586330a96B31`](https://sepolia.explorer.mode.network/address/0x2AD850d0c6b1846F9bBD8144362A586330a96B31) (impl)
```
Comment (TheDZhon) [inline docs/deployed-contracts/sepolia.md:115]: should be https://sepolia.explorer.mode.network/address/0x8Cc0183D53c8fB160BFB01fe49ff3E8A9Aa0B1F6

**Review by TheDZhon — CHANGES_REQUESTED** [review]
:eyes:

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM, but let's add the 'proposed' marks and resolve merge conflicts :pray:

**Review by TheDZhon — APPROVED** [review]
LGTM

**Review by zuzueeka — APPROVED** [review]

**Pull request merged**